plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -133,8 +134,13 @@ Job create(JobInfo jobInfo)
 
     TableResult query(String sql)
     {
+        JobInfo jobInfo = JobInfo.of(QueryJobConfiguration.of(sql));
```

Comment:
Can you explain why this fixes an issue. It is not obvious what is happening here (at least to me).

Reply:
It's library's bug. It seems recent version doesn't have this issue. I will send other PR to upgrade it instead of this workaround.